docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -5,45 +5,56 @@
 
-  Keen Python programmers may notice that contrary to PEP 8's advice we don't put ``imports`` at the top of the file. This is because the ``DOCUMENTATION`` through ``RETURN`` sections are not used by the module code itself; they are essentially extra docstrings for the file. The imports are placed after these special variables for the same reason as PEP 8 puts the imports after the introductory comments and docstrings. This keeps the active parts of the code together and the pieces which are purely informational apart. The decision to exclude E402 is based on readability (which is what PEP 8 is about). Documentation strings in a module are much more similar to module level docstrings, than code, and are never utilized by the module itself. Placing the imports below this documentation and closer to the code, consolidates and groups all related code in a congruent manner to improve readability, debugging and understanding.
+If you see any discrepancies in some older Ansible modules, please raise a pull request to make them satisfy these guidelines. 
 
-.. warning:: **Copy old modules with care!**
+Non-Python modules documentation
+================================
 
-  Some older Ansible modules have ``imports`` at the bottom of the file, ``Copyright`` notices with the full GPL prefix, and/or ``DOCUMENTATION`` fields in the wrong order. These are legacy files that need updating - do not copy them into new modules. Over time we are updating and correcting older modules. Please follow the guidelines on this page!
+For modules written in languages other than Python, there are two approaches to handling their documentation:
 
-.. note:: For non-Python modules you still create a ``.py`` file for documentation purposes. Starting at ansible-core 2.14 you can instead choose to create a ``.yml`` file that has the same data structure, but in pure YAML.
-          With YAML files, the examples below are easy to use by removing Python quoting and substituting ``=`` for ``:``, for example ``DOCUMENTATION = r''' ... '''`` to ``DOCUMENTATION: ...`` and removing closing quotes. :ref:`adjacent_yaml_doc`
+1. still create a ``.py`` file for documentation purposes that will contain the documentation-related sections described in this document or
+2. starting at ansible-core 2.14, you can instead create a ``.yml`` file that has the same data structure, but in pure YAML
 
+  * With YAML files, the examples below are easy to use by removing Python quoting and substituting ``=`` for ``:``, for example ``DOCUMENTATION = r''' ... '''`` to ``DOCUMENTATION: ...`` and removing closing quotes. Refer to :ref:`adjacent_yaml_doc` for details.
 
 .. _shebang:
 
 Python shebang & UTF-8 coding
-===============================
+=============================
+
+1. Begin your Ansible module with ``#!/usr/bin/python`` - this "shebang" allows ``ansible_python_interpreter`` to work.
```

Comment:
```suggestion
1. Begin your Ansible module with the ``#!/usr/bin/python`` shebang so that ``ansible_python_interpreter`` works as expected.
```

Reply:
I'm not quite sure about "works as expected". Maybe just "works" but maybe "takes effect" or maybe even "can correctly interpret the Python code as an Ansible module".

Reply:
i like just minimalist "works".  Thanks!